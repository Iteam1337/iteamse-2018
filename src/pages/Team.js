// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/TeamPage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import { filterByLocation } from '../utils/filterByLocation'
import FilterByLocation from '../components/FilterByLocation/FilterByLocation'
import TeamMembers from '../components/Team/TeamMembers'
import TeamPageMember from '../components/Team/TeamPageMember'
import Header from '../components/Header/Header'
import GridColumn from '../components/Grid/GridColumn'

type Props = Iteam.ApolloBase<IteamCMS.TeamPage>

export const TeamPageQuery = gql`
  query TeamPage {
    pageTeam {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
    }
    team {
      avatar
      email
      location
      name
      phoneNumber
      short
      title
    }
  }
`

const Team = () => {
  return (
    <Query query={TeamPageQuery}>
      {({ loading, data: { pageTeam, team } }: Props) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageTeam.headerImage}
              messageBgColor={pageTeam.headerTextBgColor}
              messageOne={pageTeam.headerText1}
              messageTwo={pageTeam.headerText2}
            />

            <GridColumn>
              <FilterByLocation>
                {location => (
                  <TeamMembers bgColor="#fff" teamMembers={team.length}>
                    {team
                      .filter(filterByLocation(location))
                      .map(member => (
                        <TeamPageMember key={member.name} member={member} />
                      ))}
                  </TeamMembers>
                )}
              </FilterByLocation>
            </GridColumn>
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default Team
