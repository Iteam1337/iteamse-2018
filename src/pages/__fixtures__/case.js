export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
