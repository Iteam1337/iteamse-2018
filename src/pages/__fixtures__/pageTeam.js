export const pageTeam = {
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/4l7Q40idLOOI6MCscS8o2C/d95c6f40226fd707eb692fdd806acdef/team.jpg',
  headerText1: 'Det är vi som är Iteam',
  headerText2: null,
  headerTextBgColor: '#668CFF',
  __typename: 'PageTeam',
}
