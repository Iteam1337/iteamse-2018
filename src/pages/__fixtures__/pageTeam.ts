export const pageTeam = {
  __typename: 'PageTeam',
  contactTitle: 'Hör av dig till oss!',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/4l7Q40idLOOI6MCscS8o2C/d95c6f40226fd707eb692fdd806acdef/team.jpg',
  headerText1: 'Det är vi som är Iteam',
  headerText2: null,
  headerTextBgColor: '#668CFF',
  team: ['rln', 'msr'],
}
