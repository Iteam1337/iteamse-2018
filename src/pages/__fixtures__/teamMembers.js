export const teamMembers = [
  {
    avatar: '',
    email: 'cm@sesame.com',
    gravatar: 'cm.png',
    location: 'Stockholm',
    name: 'Cookie M',
    phoneNumber: '0701234567',
    title: 'Cookie master',
    short: 'cmm',
    __typename: 'TeamMember',
  },
  {
    avatar: '',
    email: 'count@sesame.com',
    gravatar: 'count.png',
    location: 'GÃ¶teborg',
    name: 'Count van Count',
    phoneNumber: '0707654321',
    title: '1 ahahah',
    short: 'cvc',
    __typename: 'TeamMember',
  },
]
