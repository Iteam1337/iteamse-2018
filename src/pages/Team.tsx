import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import { Helmet } from 'react-helmet'
import { TeamPageQuery } from '../../typings/iteamse'
import FilterByLocation from '../components/FilterByLocation/FilterByLocation'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import Team from '../components/Team/Team'
import TeamMembers from '../components/Team/TeamMembers'
import TeamPageMember from '../components/Team/TeamPageMember'
import { filterByLocation } from '../utils/filterByLocation'
import { set } from '../utils/googleAnalytics'

export const TEAM_PAGE_QUERY = gql`
  query TeamPage {
    pageTeam {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      contactTitle
      team
    }
    team {
      avatar
      email
      location
      name
      phoneNumber
      short
      title
    }
  }
`

class TeamQuery extends Query<TeamPageQuery> {}

export class TeamPage extends React.Component {
  componentDidMount() {
    set('teamet')
  }

  render() {
    return (
      <>
        <Helmet>
          <title>Iteam - There's a better way |Â Teamet</title>
        </Helmet>
        <TeamQuery query={TEAM_PAGE_QUERY}>
          {({ loading, data }) => {
            if (loading || !data) {
              return null
            }

            const { pageTeam, team } = data

            return (
              <React.Fragment>
                <Header
                  backgroundImage={pageTeam.headerImage}
                  messageBgColor={pageTeam.headerTextBgColor}
                  messageOne={pageTeam.headerText1}
                  messageTwo={pageTeam.headerText2}
                />

                <GridColumn>
                  <FilterByLocation>
                    {location => (
                      <TeamMembers bgColor="#fff" teamMembers={team.length}>
                        {team
                          .filter(filterByLocation(location))
                          .map(member => (
                            <TeamPageMember key={member.name} member={member} />
                          ))}
                      </TeamMembers>
                    )}
                  </FilterByLocation>
                </GridColumn>

                <Team
                  bgColor="red"
                  callToAction={pageTeam.contactTitle}
                  shortName={pageTeam.team}
                />
              </React.Fragment>
            )
          }}
        </TeamQuery>
      </>
    )
  }
}

export default TeamPage
