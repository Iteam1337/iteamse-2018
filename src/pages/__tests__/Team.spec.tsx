import React from 'react'
import { render, Simulate, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageTeam } from '../__fixtures__/pageTeam'
import { teamMembers } from '../__fixtures__/teamMembers'
import Team, { TEAM_PAGE_QUERY } from '../Team'

const mocks = [
  {
    request: {
      query: TEAM_PAGE_QUERY,
    },
    result: {
      data: {
        pageTeam,
        team: teamMembers,
      },
    },
  },
]

describe('components/Team', () => {
  it('renders Team', async () => {
    const { container } = render(
      <MockedQuery mocks={mocks}>
        <Team />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })

  it('should handle changing location using mouse', async () => {
    const { getByText, container } = render(
      <MockedQuery mocks={mocks}>
        <Team />
      </MockedQuery>
    )

    await wait()

    Simulate.click(getByText('Stockholm'))

    expect(container).toMatchSnapshot()
  })

  it('should handle changing location using keyboard', async () => {
    const { getByText, container } = render(
      <MockedQuery mocks={mocks}>
        <Team />
      </MockedQuery>
    )

    await wait()

    Simulate.keyUp(getByText('Göteborg'), {
      keyCode: 13,
    })

    expect(container).toMatchSnapshot()
  })

  it('should not change using keyboard if not enter or space key', async () => {
    const { getByText, container } = render(
      <MockedQuery mocks={mocks}>
        <Team />
      </MockedQuery>
    )

    await wait()

    Simulate.keyUp(getByText('Göteborg'), {
      keyCode: 9,
    })

    expect(container).toMatchSnapshot()
  })
})
