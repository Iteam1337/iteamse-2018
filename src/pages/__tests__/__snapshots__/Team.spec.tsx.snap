// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Team renders Team 1`] = `
<div>
  <div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-kEYyzF cSkcD"
    >
      <div
        class="Header__Content-sc-1ofdv2r-0 limgqW"
      >
        <div
          class="Navigation__Wrap-sc-1i56sr4-0 cGavfh"
        >
          <a
            class="sc-jKJlTe bIUarX"
          >
            <img
              data-testid="logo"
              path="test-file-stub"
              class="sc-eNQAEJ gYmRno"
            />
          </a>
          <div
            class="Navigation__NavigationItems-sc-1i56sr4-2 gfiCRw"
          >
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Erbjudanden
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Våra case
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="Navigation__Indicator-sc-1i56sr4-1 Vplmv"
            />
          </div>
        </div>
        <div
          class="Header__MessageRow-sc-1ofdv2r-1 indJOe"
        >
          <h1
            class="H1-sc-1k64xg2-0 sc-kkGfuU hAhygW"
          >
            <span
              class="Header__MessageLine-sc-1ofdv2r-2 hcZgwB"
            >
              Det är vi som är Iteam
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bwzfXH hpcCKS"
  >
    <div
      class="PaddedRow-sc-13wt5gj-0 sc-bdVaJa HChwM"
    >
      <div
        data-test="filter-"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 euGVvx"
      >
        Alla
      </div>
      <div
        data-test="filter-Stockholm"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Stockholm
      </div>
      <div
        data-test="filter-Göteborg"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Göteborg
      </div>
    </div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bZQynM svAwu"
    >
      <div
        class="TeamMembers__Members-sc-1skewny-0 knmrbX"
      >
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Cookie M"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Cookie M
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            Cookie master
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0701234567"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-123 45 67
            </a>
          </div>
          <a
            href="mailto:cm@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            cm@sesame.com
          </a>
        </div>
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Count van Count"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Count van Count
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            1 ahahah
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0707654321"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-765 43 21
            </a>
          </div>
          <a
            href="mailto:count@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            count@sesame.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`components/Team should handle changing location using keyboard 1`] = `
<div>
  <div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-kEYyzF cSkcD"
    >
      <div
        class="Header__Content-sc-1ofdv2r-0 limgqW"
      >
        <div
          class="Navigation__Wrap-sc-1i56sr4-0 cGavfh"
        >
          <a
            class="sc-jKJlTe bIUarX"
          >
            <img
              data-testid="logo"
              path="test-file-stub"
              class="sc-eNQAEJ gYmRno"
            />
          </a>
          <div
            class="Navigation__NavigationItems-sc-1i56sr4-2 gfiCRw"
          >
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Erbjudanden
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Våra case
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="Navigation__Indicator-sc-1i56sr4-1 Vplmv"
            />
          </div>
        </div>
        <div
          class="Header__MessageRow-sc-1ofdv2r-1 indJOe"
        >
          <h1
            class="H1-sc-1k64xg2-0 sc-kkGfuU hAhygW"
          >
            <span
              class="Header__MessageLine-sc-1ofdv2r-2 hcZgwB"
            >
              Det är vi som är Iteam
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bwzfXH hpcCKS"
  >
    <div
      class="PaddedRow-sc-13wt5gj-0 sc-bdVaJa HChwM"
    >
      <div
        data-test="filter-"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 euGVvx"
      >
        Alla
      </div>
      <div
        data-test="filter-Stockholm"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Stockholm
      </div>
      <div
        data-test="filter-Göteborg"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Göteborg
      </div>
    </div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bZQynM svAwu"
    >
      <div
        class="TeamMembers__Members-sc-1skewny-0 knmrbX"
      >
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Cookie M"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Cookie M
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            Cookie master
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0701234567"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-123 45 67
            </a>
          </div>
          <a
            href="mailto:cm@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            cm@sesame.com
          </a>
        </div>
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Count van Count"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Count van Count
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            1 ahahah
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0707654321"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-765 43 21
            </a>
          </div>
          <a
            href="mailto:count@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            count@sesame.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`components/Team should handle changing location using mouse 1`] = `
<div>
  <div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-kEYyzF cSkcD"
    >
      <div
        class="Header__Content-sc-1ofdv2r-0 limgqW"
      >
        <div
          class="Navigation__Wrap-sc-1i56sr4-0 cGavfh"
        >
          <a
            class="sc-jKJlTe bIUarX"
          >
            <img
              data-testid="logo"
              path="test-file-stub"
              class="sc-eNQAEJ gYmRno"
            />
          </a>
          <div
            class="Navigation__NavigationItems-sc-1i56sr4-2 gfiCRw"
          >
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Erbjudanden
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Våra case
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="Navigation__Indicator-sc-1i56sr4-1 Vplmv"
            />
          </div>
        </div>
        <div
          class="Header__MessageRow-sc-1ofdv2r-1 indJOe"
        >
          <h1
            class="H1-sc-1k64xg2-0 sc-kkGfuU hAhygW"
          >
            <span
              class="Header__MessageLine-sc-1ofdv2r-2 hcZgwB"
            >
              Det är vi som är Iteam
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bwzfXH hpcCKS"
  >
    <div
      class="PaddedRow-sc-13wt5gj-0 sc-bdVaJa HChwM"
    >
      <div
        data-test="filter-"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 euGVvx"
      >
        Alla
      </div>
      <div
        data-test="filter-Stockholm"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Stockholm
      </div>
      <div
        data-test="filter-Göteborg"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Göteborg
      </div>
    </div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bZQynM svAwu"
    >
      <div
        class="TeamMembers__Members-sc-1skewny-0 knmrbX"
      >
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Cookie M"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Cookie M
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            Cookie master
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0701234567"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-123 45 67
            </a>
          </div>
          <a
            href="mailto:cm@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            cm@sesame.com
          </a>
        </div>
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Count van Count"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Count van Count
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            1 ahahah
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0707654321"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-765 43 21
            </a>
          </div>
          <a
            href="mailto:count@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            count@sesame.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`components/Team should not change using keyboard if not enter or space key 1`] = `
<div>
  <div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-kEYyzF cSkcD"
    >
      <div
        class="Header__Content-sc-1ofdv2r-0 limgqW"
      >
        <div
          class="Navigation__Wrap-sc-1i56sr4-0 cGavfh"
        >
          <a
            class="sc-jKJlTe bIUarX"
          >
            <img
              data-testid="logo"
              path="test-file-stub"
              class="sc-eNQAEJ gYmRno"
            />
          </a>
          <div
            class="Navigation__NavigationItems-sc-1i56sr4-2 gfiCRw"
          >
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Erbjudanden
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Våra case
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-hMqMXs fILlfv"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="Navigation__Indicator-sc-1i56sr4-1 Vplmv"
            />
          </div>
        </div>
        <div
          class="Header__MessageRow-sc-1ofdv2r-1 indJOe"
        >
          <h1
            class="H1-sc-1k64xg2-0 sc-kkGfuU hAhygW"
          >
            <span
              class="Header__MessageLine-sc-1ofdv2r-2 hcZgwB"
            >
              Det är vi som är Iteam
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bwzfXH hpcCKS"
  >
    <div
      class="PaddedRow-sc-13wt5gj-0 sc-bdVaJa HChwM"
    >
      <div
        data-test="filter-"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 euGVvx"
      >
        Alla
      </div>
      <div
        data-test="filter-Stockholm"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Stockholm
      </div>
      <div
        data-test="filter-Göteborg"
        tabindex="0"
        class="Filter__Wrap-sc-3nf4pp-0 kRhkKn"
      >
        Göteborg
      </div>
    </div>
    <div
      class="GridColumn__GridColumnClean-sc-1xz61ww-0 sc-bZQynM svAwu"
    >
      <div
        class="TeamMembers__Members-sc-1skewny-0 knmrbX"
      >
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Cookie M"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Cookie M
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            Cookie master
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0701234567"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-123 45 67
            </a>
          </div>
          <a
            href="mailto:cm@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            cm@sesame.com
          </a>
        </div>
        <div
          class="Colleague-fj37xp-0 jRaYiQ"
        >
          <a
            class="sc-htpNat kDzZbW"
          >
            <div
              class="TeamPageMember__AvatarWrap-sc-1a8kswp-2 kzBeRL"
            >
              <div>
                <img
                  alt="Count van Count"
                  src=""
                  class="Avatar__AvatarImage-sc-1v5vzph-0 fqhskt"
                />
              </div>
            </div>
          </a>
          <div
            class="TeamPageMember__Name-sc-1a8kswp-0 iYiJif"
          >
            Count van Count
          </div>
          <div
            class="TeamPageMember__Title-sc-1a8kswp-1 dqGprz"
          >
            1 ahahah
          </div>
          <div
            class="TeamPageMember__PhoneNumberWrap-sc-1a8kswp-3 bfobLH"
          >
            <a
              href="tel:0707654321"
              class="NativeLink-sc-1sy7t3m-0 sc-EHOje goTsLn"
            >
              070-765 43 21
            </a>
          </div>
          <a
            href="mailto:count@sesame.com"
            class="NativeLink-sc-1sy7t3m-0 sc-ifAKCX cASXVu"
          >
            count@sesame.com
          </a>
        </div>
      </div>
    </div>
  </div>
</div>
`;
